src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckJavaAgent.kt

```diff
@@ -278,18 +298,17 @@ internal object LincheckJavaAgent {
                 ensureObjectIsTransformed(it, processedObjects)
             }
         clazz.superclass?.let {
-            if (it.name in instrumentedClassesInTheModelCheckingMode) return // already instrumented
+            if (it.name in instrumentedClasses) return // already instrumented
             ensureClassHierarchyIsTransformed(it, processedObjects)
         }
     }
 
     /**
      * FOR TEST PURPOSE ONLY!
-     * To test the byte-code transformation correctness for the
-     * model-checking strategy, we can transform all classes.
+     * To test the byte-code transformation correctness, we can transform all classes.
      */
-    internal val INSTRUMENT_ALL_CLASSES_IN_MODEL_CHECKING_MODE =
-        System.getProperty("lincheck.instrumentAllClassesInModelCheckingMode")?.toBoolean() ?: false
+    internal val INSTRUMENT_ALL_CLASSES =
+        System.getProperty("lincheck.instrumentAllClasses")?.toBoolean() ?: false
```

Comment:
Could you please remind me why the property was renamed?

Reply:
Because now stress strategy also has an optimization when it does not re-transform all the classes each time, but only:
1. subset of previously seen classes that have coroutine calls;
2. new unseen classes that were loaded into VM since the last re-transformation.

This flag now disables class re-transformation optimizations for both strategies, enforcing them to always re-transform all the loaded classes.